.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+    'root': true,
+    'env': {
+        'node': true
+    },
+    'extends': [
+        'eslint:recommended',
+        'plugin:unicorn/recommended',
```

Comment:
What is this unicorn plugin? It does not seem to be present in the original tslint.json so I don't understand why it's present here.